### PR TITLE
[INJICERT-434] update Mimoto issuers to add another Certify+eSignet instance

### PR DESCRIPTION
**Pull request by vharsh**
**Configuration**:

* eSignet 1.4.1 from eSignet namespace
* Certify from inji-certify namespace

### CODE REVIEW

**Review by jainhitesh9998 — APPROVED** [review]

File: mimoto-issuers-config.json [hunk 1]
```diff
@@ -231,6 +231,44 @@
...
+        "client_alias": "mpartner-mock-testing",
+        "scopes_supported": [
+          "mock_identity_vc_ldp"
+        ],
+        "additional_headers": {
+          "Accept": "application/json"
+        },
+        "wellknown_endpoint": "https://${mosip.injicertify.host}/v1/certify/issuance/.well-known/openid-credential-issuer",
+        "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+        "authorization_endpoint": "https://${mosip.esignet.mock.host}/authorize",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:259]: remove this as it's being referred from wellknown

File: mimoto-issuers-config.json [hunk 2]
```diff
@@ -231,6 +231,44 @@
...
+        ],
+        "protocol": "OpenId4VCI",
+        "client_id": "${mimoto.oidc.mosipid.partner.clientid}",
+        "client_alias": "mpartner-mock-testing",
+        "scopes_supported": [
+          "mock_identity_vc_ldp"
+        ],
+        "additional_headers": {
+          "Accept": "application/json"
+        },
```
Comment (swatigoel) [inline mimoto-issuers-config.json:256]: remove this as it's not in use

File: mimoto-issuers-config.json [hunk 3]
```diff
@@ -231,6 +231,44 @@
...
+            "description": "Download Mock Identity Credential via DataProvider Plugin Impl.",
+            "language": "en"
+          }
+        ],
+        "protocol": "OpenId4VCI",
+        "client_id": "${mimoto.oidc.mosipid.partner.clientid}",
+        "client_alias": "mpartner-mock-testing",
+        "scopes_supported": [
+          "mock_identity_vc_ldp"
+        ],
```
Comment (swatigoel) [inline mimoto-issuers-config.json:253]: remove this as it's being referred from wellknown

File: mimoto-issuers-config.json [hunk 4]
```diff
@@ -231,6 +231,44 @@
...
+        "authorization_endpoint": "https://${mosip.esignet.mock.host}/authorize",
+        "authorization_audience": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
+        "token_endpoint": "https://${mosip.api.public.host}/residentmobileapp/get-token/MockCertifyDP",
+        "proxy_token_endpoint": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
+        "credential_endpoint": "https://${mosip.injicertify.host}/v1/certify/issuance/credential",
+        "credential_type": [
+          "VerifiableCredential",
+          "MOCKVerifiableCredential_ldp"
+        ],
+        "credential_audience": "https://${mosip.injicertify.host}",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:268]: remove these as it's being referred from wellknown

File: mimoto-issuers-config.json [hunk 5]
```diff
@@ -231,6 +231,44 @@
       "qr_code_type": "OnlineSharing",
       "enabled": "true"
     },
+    {
+        "credential_issuer": "MockCertifyDP",
+        "display": [
+          {
+            "name": "Mock Identity (Certify) DataProvider",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:238]: Name should be `Mock Identity DataProvider`

File: mimoto-issuers-config.json [hunk 6]
```diff
@@ -231,6 +231,44 @@
...
+              "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
+              "alt_text": "mosip-logo"
+            },
+            "title": "Mock Identity (Certify) DataProvider",
+            "description": "Download Mock Identity Credential via DataProvider Plugin Impl.",
+            "language": "en"
+          }
+        ],
+        "protocol": "OpenId4VCI",
+        "client_id": "${mimoto.oidc.mosipid.partner.clientid}",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:249]: use mock server clientId

**Pull request merged**